Iterator/TransformIterator.php

```diff
@@ -28,33 +28,35 @@ public function __construct($iterator, $transform)
         $this->transform = $transform;
     }
 
+    #[\ReturnTypeWillChange]
```

Comment:
This is needed, as we can't use `: mixed` as return type hint, as that is not supported by PHP 7.2. The attribute comment will suppress the warning in PHP 8.1